back/spec/acceptance/ideas_spec.rb

```diff
@@ -1198,6 +1202,10 @@
             expect(response_data.dig(:relationships, :author, :data, :id)).to eq @user.id
             expect(response_data.dig(:attributes, :anonymous)).to be false
           end
+
```

Comment:
Do we need another test to check that it also removes any values for ideas already posted?

Reply:
Sounds good :) I didn't think of that.